### PR TITLE
Update EC2 MC procedure

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md [hunk 1]
```diff
@@ -5,107 +5,170 @@ title: Amazon EC2
...
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Aws-Ec2-Instance-custom** est utilisé.
 
 </TabItem>
-<TabItem value="Services" label="Services">
+<TabItem value="Cloud-Aws-Ec2-Spot-Fleet-Request-custom" label="Cloud-Aws-Ec2-Spot-Fleet-Request-custom">
 
-Aucune règle de découverte de service n'est associée à ce connecteur de supervision. 
+| Alias                    | Modèle de service                   | Description                                                                                   |
+|:-------------------------|:------------------------------------|:----------------------------------------------------------------------------------------------|
+| EC2Spot-Active-Instances | Cloud-Aws-Ec2-Spot-Instances-custom | Contrôle permettant de contrôler le nombre d'instance spot active pour une Spot Fleet Request |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md:53]:
```suggestion
| EC2Spot-Active-Instances | Cloud-Aws-Ec2-Spot-Instances-custom | Contrôle permettant de contrôler le nombre d'instances spot active pour une Spot Fleet Request |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md [hunk 2]
```diff
@@ -5,107 +5,170 @@ title: Amazon EC2
...
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Aws-Ec2-Spot-Fleet-Request-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                       | Modèle de service                                | Description                                                     |
+|:----------------------------|:-------------------------------------------------|:----------------------------------------------------------------|
+| Ec2-Asg-Status-Global       | Cloud-Aws-Ec2-Asg-Status-Global-Api-custom       | Contrôle permettant de vérifier le statut des ASG               |
+| Ec2-Instances-Status-Global | Cloud-Aws-Ec2-Instances-Status-Global-Api-custom | Contrôle permettant de vérifier le statut des instances         |
+| Ec2-Instances-Types-Global  | Cloud-Aws-Ec2-Instances-Types-Global-Api-custom  | Contrôle permettant de remonter le nombre d'instances par types |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md:64]:
```suggestion
| Ec2-Instances-Types-Global  | Cloud-Aws-Ec2-Instances-Types-Global-Api-custom  | Contrôle permettant de remonter le nombre d'instances par type |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md [hunk 3]
```diff
@@ -5,107 +5,170 @@ title: Amazon EC2
...
+| ec2.instances.status.stopping.count     | count |
+| ec2.instances.status.stopped.count      | count |
+| *aws_instances*#status                  | N/A   |
+| *aws_instances*#ec2.uptime.days         | d     |
+
+</TabItem>
+<TabItem value="Ec2-Instances-Types-Global" label="Ec2-Instances-Types-Global">
 
-> **Remarque** Ces métriques sont généralement nulles lorsque collectées sur un groupe d'AutoScaling.
+Aucune métrique pour ce modèle.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md:156]: commentaire intéressant en face?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md [hunk 4]
```diff
@@ -5,107 +5,170 @@ title: Amazon EC2
...
- * Instance-Types: Nombre d'instances pour chaque famille d'instance AWS/EC2 et types associés.
- * Instance-Status: Statuts unitaires et globaux de vos instances EC2 en cours d'exécution (ainsi que le nombre total)
+Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.
 
 <Tabs groupId="sync">
-<TabItem value="Ec2-Cpu-Credit" label="Ec2-Cpu-Credit">
+<TabItem value="EC2Spot-Active-Instances" label="EC2Spot-Active-Instances">
 
-Ce contrôle est associé aux Modèles d'Hôtes suivants: 'Cloud-Aws-Ec2-Asg' et 'Cloud-Aws-Ec2-Instance'.
+| Métrique                           | Unité |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md:89]: Why did all the descriptions disappear?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md [hunk 5]
```diff
@@ -124,9 +187,9 @@ Voici la liste des droits nécessaires au travers des access/secret key utilisé
 
 ### Dépendances du Plugin
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md:188]:
```suggestion
### Dépendances du plugin
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md [hunk 6]
```diff
@@ -5,107 +5,170 @@ title: Amazon EC2
...
+| Métrique                   | Unité |
+|:---------------------------|:------|
+| ec2.disk.bytes.read.bytes  | B     |
+| ec2.disk.bytes.write.bytes | B     |
+| ec2.disk.ops.read.count    | ops   |
+| ec2.disk.ops.write.count   | ops   |
+| ec2.disk.bytes.read.bytes  | B     |
+| ec2.ebs.bytes.write.bytes  | B     |
+| ec2.ebs.ops.read.count     | ops   |
+| ec2.ebs.ops.write.count    | ops   |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-ec2.md:138]:
```suggestion
| ec2.ebs.ops.write.count    | ops   |

> **Remarque** Ces métriques sont généralement nulles lorsque collectées sur un groupe d'AutoScaling.

```

File: pp/integrations/plugin-packs/procedures/cloud-aws-ec2.md [hunk 7]
```diff
@@ -5,111 +5,174 @@ title: Amazon EC2
...
+| Metric name                | Unit  |
+|:---------------------------|:------|
+| ec2.disk.bytes.read.bytes  | B     |
+| ec2.disk.bytes.write.bytes | B     |
+| ec2.disk.ops.read.count    | ops   |
+| ec2.disk.ops.write.count   | ops   |
+| ec2.disk.bytes.read.bytes  | B     |
+| ec2.ebs.bytes.write.bytes  | B     |
+| ec2.ebs.ops.read.count     | ops   |
+| ec2.ebs.ops.write.count    | ops   |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-aws-ec2.md:136]:
```suggestion
| ec2.ebs.ops.write.count    | ops   |

> These metrics are most of the time irrelevant or null when applied to an AutoscalingGroup

```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**